src/api/Nhs.Appointments.Api/Functions/SetBookingStatusFunction.cs

```diff
@@ -32,7 +32,7 @@ public class SetBookingStatusFunction : BaseApiFunction<SetBookingStatusRequest,
     [OpenApiSecurity("Api Key", SecuritySchemeType.ApiKey, Name = "Authorization", In = OpenApiSecurityLocationType.Header)]
     [OpenApiResponseWithBody(statusCode:HttpStatusCode.OK, "text/plain", typeof(SetBookingStatusResponse), Description = "Booking status successfully set")]
     [OpenApiResponseWithBody(statusCode:HttpStatusCode.NotFound, "text/json", typeof(ApiResult<object>), Description = "Booking not found")]
-    [RequiresPermission("booking:set-status", typeof(NoSiteRequestInspector))]
+    [RequiresPermission("booking:set-status", typeof(SiteFromBodyInspector))]
```

Comment:
The body for this request does not contain site, so does this work?

Reply:
The request from the frontend includes the site in client/src/services/bookingservice.ts, which is then picked up by the SiteFromBodyInspector's JSON parsing, so its not in the SetBookingStatusRequest but is only used to determine authorisation